### PR TITLE
Ajust gps aligment of Status and Quality

### PR DESCRIPTION
**Pull request by HThuren**
New method to align Status and Quality, as wanted by @McGiverGim in #3566 

Size fit to other languages than english.

image

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by nerdCopter — APPROVED** [review]
approving based on screencap. not tested.

**Pull request merged**
